### DISCUSSION TITLE
github.dev search doesn't work in Edge

### DISCUSSION THREAD

**Discussion by GuardianDll** [community/community#62290] [Codespaces]
### Select Topic Area

Bug

### Body

Got used to new github.dev editor, however, for some reason, the search bar doesn't work for me, no matter of settings or prompt
![image](https://github.com/community/community/assets/67688115/9a89233b-4ef2-45b0-b2f0-9e89cb800c3c)

I believe the reason is because i use Edge browser, attempt to use .dev in chrome works as expected
![image](https://github.com/community/community/assets/67688115/7872171a-09b4-48a2-b8c1-b3cde3310b65)

**Comment by gorgeousgeorge7** [comment]
Clear Browser Cache: Sometimes browser cache and cookies can cause unexpected issues. Try clearing your browser's cache and cookies and then refreshing the page.

Disable Browser Extensions: Browser extensions or add-ons can sometimes interfere with website functionality. Try disabling any browser extensions and see if the search bar starts working.

Use a Different Browser: Try accessing GitHub.dev using a different web browser to see if the issue is specific to the browser you're using.

Check Network Connection: Ensure that your internet connection is stable. Poor network connectivity can lead to loading issues and non-functional features.

Update Browser: Make sure you're using the latest version of your web browser. Outdated browsers might have compatibility issues with newer web technologies.

GitHub.dev Updates: The GitHub.dev editor is under active development, and there might be occasional bugs or issues. Make sure you're using the latest version of GitHub.dev, and if the issue persists, consider reporting it to GitHub support or through their feedback channels.

Browser Compatibility: Ensure that GitHub.dev is compatible with your browser. Some features might not work properly on all browsers.

JavaScript and Security Settings: Check if JavaScript is enabled in your browser and that your browser's security settings are not overly restrictive, as these settings can affect the functionality of web applications.

GitHub Status: Check GitHub's status page (https://www.githubstatus.com/) to see if there are any ongoing issues with GitHub services that might be affecting the search functionality.
